README.md

```diff
@@ -14,24 +14,54 @@ Add/write sourcemaps to/from Vinyl files.
 
 ```js
 sourcemap.add(file, function(err, updatedFile) {
-
+  // updatedFile will have a .sourceMap property
 });
 
+// The 2nd argument can be given as a path string
 sourcemap.write(file, './maps', function(err, updatedFile, sourcemapFile) {
+  // sourcemapFile will be a Vinyl file to be written to some location
```

Comment:
Looks good to me. However, you might want to move section 15-24 to after line 58. I recommend this as the below explanations are much more thorough. The below makes it clear that all the files are Vinyl file objects where only line 22 above makes that explicit.

Anyway, if you like it the way it is then merge away.